todo_app/data/trello_items.py

```diff
@@ -0,0 +1,46 @@
+import requests
+import os
+
+API_KEY = os.getenv('TRELLO_API_KEY')
+TOKEN = os.getenv('TRELLO_TOKEN')
+BOARD_ID = os.getenv('TRELLO_BOARD_ID')
+BASE_URL = "https://api.trello.com/1/"
+
+def convert_trello_list(trello_list):
+    items = []
+    for card in trello_list['cards']:
+        items.append(Item.from_trello_card(card, trello_list))
+    return items
+
+
+def get_to_do_items():
+    to_do_list = requests.get(f"{BASE_URL}boards/{BOARD_ID}/lists?key={API_KEY}&token={TOKEN}&cards=open").json()[0]
```

Comment:
Note that requests lets you pass in params this way too:

```
params = {'key': API_KEY, 'token': TOKEN, 'cards': 'open' }
requests.get(f'{BASE_URL}boards/{BOARD_ID}/lists', params=params)
```

Which is a little bit nicer. Plus you could have a re-usable auth params dictionary that individual functions can add to.